### DISCUSSION TITLE
Is there a way to whitelist domains and IPs?

### DISCUSSION THREAD

**Discussion by Your-New-SJW-Waifu** [qdm12/gluetun#1391] [Q&A]
I have some local stuff I access over http plus some remote stuff that I don't want to send through http proxy. 

Is there a way to setup a whitelist for domains and IPs/subnets so that when a matching IP/domain is requested, it doesn't get routed through the VPN? 

Ideally I could even use a different DNS server to resolve domains on the whitelist.

**Comment by woodfairy** [comment]
I am also interested in this, is there any way achieving this?

**Comment by Marcbacca** [comment]
I guess no answer in 2 years means no .. just searching this as well.
